wallet/src/Pos/Wallet/Web/Api.hs

```diff
@@ -386,6 +394,8 @@ type WalletApi = ApiPrefix :> (
      -- NOTE: enabled in prod mode https://issues.serokell.io/issue/CSM-333
      TestReset
     :<|>
+     TestState
```

Comment:
Surely there is some mechanism which does this automatically, but how can we prevent that this endpoint will leak to production as it's for debugging-purposes only?

Reply:
we can add flag `--with-test-endpoints` which will include these `/test` endpoints and turn off it in production